dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -1678,6 +1683,53 @@ internal class RumViewScopeTest {
         verifyNoMoreInteractions(mockDevLogHandler)
     }
 
+    @Test
+    fun `𝕄 send action 𝕎 handleEvent(StartAction) + active child ActionScope { custom action }`(
```

Comment:
Nice test, can we add one when `waitForStop = true` and `type = RumActionType.CUSTOM`, just to ensure this case will still work as expected